applications/launchpad/gui-react/src/locales/mining.ts

```diff
@@ -28,6 +28,14 @@ const translations = {
       'Tari Launchpad must be open at the scheduled hours for mining to start.',
     noSchedules: 'No mining schedule has been set up yet',
     add: 'Add schedule',
+    removeSchedule: 'Remove schedule',
+    ops: 'Ops!',
+    error_miningEndsBeforeItStarts:
+      /* eslint-disable-next-line quotes */
+      "I guess you need to correct the hours because mining can't stop before it even starts",
+    error_miningEndsWhenItStarts:
+      /* eslint-disable-next-line quotes */
```

Comment:
NIT: Maybe we can move this eslint rule at the beginning so it disables that for whole file.